Example/Auth/Sample/ApplicationDelegate.m

```diff
@@ -17,9 +17,9 @@
 #import "ApplicationDelegate.h"
 
 #import "AuthProviders.h"
-#import "FirebaseCommunity/FIRApp.h"
+#import "FirebaseCore/FIRApp.h"
 #import "FirebaseAuth.h"
-#import "FirebaseCommunity/FIRLogger.h"
+#import "FirebaseCore/FIRLogger.h"
```

Comment:
Running `scripts/style.sh` should reorder these imports (unless Auth exempts itself).

Reply:
Auth is exempt, but will run style.sh for rest